iaac/terraform/kubeflow-components/kubeflow-pipelines/variables.tf

```diff
@@ -0,0 +1,31 @@
+variable "helm_config" {
+  type        = any
+  default     = {}
+}
+variable rds_db_host {
```

Comment:
only 2 variables, this is for later not this pr

Reply:
done